client/me/help/help-contact/index.jsx

```diff
@@ -389,14 +389,21 @@ const HelpContact = React.createClass( {
 	},
 
 	shouldUseHappychat: function() {
+		const { olark } = this.state;
 		const { isHappychatAvailable } = this.props;
 		const { olark: { isUserEligible } } = this.state;
```

Comment:
I would remove this line since olark is pulled in above, and change below to `olark.isUserEligible`